source/server/overload_manager_impl.cc

```diff
@@ -86,14 +121,29 @@ class ThreadLocalOverloadStateImpl : public ThreadLocalOverloadState {
     return always_inactive_;
   }
 
+  Event::TimerPtr createScaledTimer(OverloadTimerType timer_type,
+                                    Event::TimerCb callback) override {
+    auto minimum_it = timer_minimums_.find(timer_type);
+    ScaledTimerMinimum minimum =
+        minimum_it != timer_minimums_.end() ? minimum_it->second : ScaledTimerMinimum(1.0);
+    return std::make_unique<FixedMinimumScaledTimer>(
+        scaled_timer_manager_->createTimer(std::move(callback)), minimum);
```

Comment:
The default minimum is ScaledTimerMinimum(1.0) which makes min = max. It seemed more straightforward to avoid special casing, though the behavior of the returned range timer is degenerate and it triggers exactly when its wrapper TimerPtr triggers.